phpcs.xml

```diff
@@ -2,7 +2,7 @@
 <ruleset name="10up PHPCS">
 	<description>10up PHPCS extended.</description>
 	<exclude-pattern>dist/</exclude-pattern>
-	<exclude-pattern>plugins/*</exclude-pattern>
+	<exclude-pattern>./plugins/*</exclude-pattern>
```

Comment:
Is it possible to add an override for a exclude? 

In that case I would like to say somthing like this: (pseudo code)

```xml
    <exclude-pattern>./plugins/*</exclude-pattern>
    <exclude-pattern>./mu-plugins/*</exclude-pattern>
    <include-pattern>./mu-plugins/tenup-plugin</include-pattern>
```

So that any other installed 3rd party MU plugins don't get checked. 

Essentially the goal should be to only check first party code 🤔 I would kind of want this to just use the `.gitignore` rules.

Reply:
Just noting that `<include-pattern>` can only be used inside specific rules. We should use the `<file>` directive to add specific directories, though it seems that `<exclude-pattern>` will override those added via `<file>`.

It may be that we need to use some convoluted regex to allow specific directories if we blanket ignore `./plugins/*`